archive/c/c-plus-plus/longest-common-subsequence.cpp

```diff
@@ -0,0 +1,107 @@
+#include<iostream>
+#include<algorithm>
+#include<vector>
+#include<string>
+using namespace std;
+
+//The function to convert the input strings into the required array, by using delimiter.
+vector<string> splitStrings(string str) 
+{   
+	string word = ""; 
```

Comment:
Indentation is a bit inconsistent here. Any chance you could run some formatting on these files?

Reply:
ok I will do that.